src/rules/noObjectMath.ts

```diff
@@ -0,0 +1,69 @@
+import { makeRule, getParserServices, getConstrainedTypeAtLocation } from "../util";
+import ts from "typescript";
+
+const dataTypes = ["CFrame", "UDim", "UDim2", "Vector2", "Vector2int16", "Vector3", "Vector3int16"];
+const opTypes = new Map([
+	["+", "add"] as const,
```

Comment:
My version of TS was making me use `as const` here. It was a bug with a nightly build. Hope this doesn't bother anyone.